bears/python/requirements/PySafetyBear.py

```diff
@@ -0,0 +1,96 @@
+from collections import namedtuple
+import pkg_resources
+import re
+
+from safety import safety
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+from coalib.results.SourceRange import SourceRange
+from coalib.settings.Setting import typed_list
+
+
+# the safety module expects an object that looks like this
+# (not importing it from there because it's in a private-ish location)
+Package = namedtuple('Package', ('key', 'version'))
+
+
+class PySafetyBear(LocalBear):
+    """
+    Checks if any of your Python dependencies have known security issues.
+
+    Data is taken from pyup.io's vulnerability database hosted at
+    https://github.com/pyupio/safety.
+    """
+
+    LANGUAGES = {
+        'Python Requirements',
+        'Python 2 Requirements',
+        'Python 3 Requirements',
+    }
+    AUTHORS = {'Bence Nagy'}
+    REQUIREMENTS = {PipRequirement('safety', '0.3.*')}
+    AUTHORS_EMAILS = {'bence@underyx.me'}
+    LICENSE = 'MIT'
```

Comment:
btw. I think the bear itself has to be AGPL, however the wrapped tool can be MIT. I was thinking about removing the license field because it could lead to confusion... other opinions?